data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -345,10 +358,17 @@ then
         --volume "$PWD:/assets:z" \
         --volume /etc/kubernetes:/etc/kubernetes:z \
         "${CLUSTER_BOOTSTRAP_IMAGE}" \
+{{- if .BootstrapInPlace}}
```

Comment:
This section could use `.BootstrapInPlace` to set a variable like `REQUIRED_PODS` and then use that variable in the command line below.

What is `assets-create-timeout` if we are not using the bootstrap-in-place approach?

Reply:
It's how long cluster-bootstrap should wait for all manifests to be created.
In a regular installation, it might take some time because some manifests might require an operator to start running and create the CRD in order to apply the CR.
In bootstrap-in-place deployment, no operator will start running so there is no reason to wait that long.

Reply:
This used to be an optimization while single node installation required patches to CVO.
I think we can go back to use the default.